openeo/extra/job_management.py

```diff
@@ -337,43 +358,39 @@ def run_jobs(
         if job_db.exists():
             # Resume from existing db
             _log.info(f"Resuming `run_jobs` from existing {job_db}")
-            df = job_db.read()
-            status_histogram = df.groupby("status").size().to_dict()
-            _log.info(f"Status histogram: {status_histogram}")
-
-        df = self._normalize_df(df)
+            #df = job_db.read()
+            #status_histogram = df.groupby("status").size().to_dict()
+            #_log.info(f"Status histogram: {status_histogram}")
+        else:
+            df = self._normalize_df(df)
+            job_db.persist(df)
```

Comment:
I like that this PR decouples the dataframe aspects from `run_jobs` and I think we can even take it a small step further by making the `df` argument optional  now:
- allow it to be `None` in function signature and docs
- only do the normalize and persist if it is defined

Reply:
done